instrumentation/apache-shenyu/apache-shenyu-2.4/javaagent/build.gradle.kts

```diff
@@ -0,0 +1,49 @@
+plugins {
+  id("otel.javaagent-instrumentation")
+}
+
+muzzle {
+  pass {
+    group.set("org.apache.shenyu")
+    module.set("shenyu-web")
+    versions.set("[2.4.0,)")
+  }
+}
+
+dependencies {
+  compileOnly("org.apache.shenyu:shenyu-web:2.4.0")
+  compileOnly("com.google.auto.value:auto-value-annotations")
+  annotationProcessor("com.google.auto.value:auto-value")
+
+  testInstrumentation(project(":instrumentation:apache-shenyu:apache-shenyu-2.4:javaagent"))
+  testInstrumentation(project(":instrumentation:netty:netty-4.1:javaagent"))
+  testInstrumentation(project(":instrumentation:reactor:reactor-3.1:javaagent"))
+  testInstrumentation(project(":instrumentation:reactor:reactor-netty:reactor-netty-1.0:javaagent"))
+  testInstrumentation(project(":instrumentation:spring:spring-webflux:spring-webflux-5.0:javaagent"))
+  testLibrary("org.springframework.boot:spring-boot-starter-test:2.0.0.RELEASE")
```

Comment:
I'd define the dependency before the parts of `testInstrumentation`. It had better to define in a same code block for same type.

Reply:
Do you mean that I should put the `testLibrary` before the `testInstrumentation` block?

Reply:
> Do you mean that I should put the `testLibrary` before the `testInstrumentation` block?

Yes

Reply:
done